doc/classes/Array.xml

```diff
@@ -297,88 +303,83 @@
 			<param index="1" name="from" type="int" default="0" />
 			<description>
-				Searches the array for a value and returns its index or [code]-1[/code] if not found. Optionally, the initial search index can be passed.
+				Returns the index of the [b]first[/b] occurrence of [param what] in this array, or [code]-1[/code] if there are none. The search's start can be specified with [param from], continuing to the end of the array.
+				[b]Note:[/b] If you just want to know whether the array contains [param what], use [method has] ([code]Contains[/code] in C#). In GDScript, you may also use the [code]in[/code] operator.
 			</description>
 		</method>
 		<method name="front" qualifiers="const">
 			<return type="Variant" />
 			<description>
-				Returns the first element of the array. Prints an error and returns [code]null[/code] if the array is empty.
-				[b]Note:[/b] Calling this function is not the same as writing [code]array[0][/code]. If the array is empty, accessing by index will pause project execution when running from the editor.
+				?? Returns the first element of the array. If the array is empty, generates an error and returns [code]null[/code], similarly to [], without stopping the project execution.
+				?? Returns the first element of the array. If the array is empty, generates an error and returns [code]null[/code] without stopping the project execution (unlike with the [code][][/code] operator).
+				?? [b]Note:[/b] This method is not the same as [code]array[0][/code].
 			</description>
 		</method>
 		<method name="get_typed_builtin" qualifiers="const">
 			<return type="int" />
 			<description>
-				Returns the [enum Variant.Type] constant for a typed array. If the [Array] is not typed, returns [constant TYPE_NIL].
+				Returns the [enum Variant.Type] constant for the typed array. If the array is not typed, returns [constant TYPE_NIL]. See also [method set_typed].
 			</description>
 		</method>
 		<method name="get_typed_class_name" qualifiers="const">
 			<return type="StringName" />
 			<description>
-				Returns a class name of a typed [Array] of type [constant TYPE_OBJECT].
+				Returns the base class name of the typed [Array] of type [constant TYPE_OBJECT]. See also [method set_typed].
 			</description>
 		</method>
 		<method name="get_typed_script" qualifiers="const">
 			<return type="Variant" />
 			<description>
-				Returns the script associated with a typed array tied to a class name.
+				Returns the custom [Script] associated with this typed array. See also [method set_typed].
 			</description>
 		</method>
 		<method name="has" qualifiers="const">
 			<return type="bool" />
 			<param index="0" name="value" type="Variant" />
 			<description>
-				Returns [code]true[/code] if the array contains the given value.
+				?? Returns [code]true[/code] if the array contains the given [param value].
+				?? Returns [code]true[/code] if any of the array's elements is the given [param value].
```

Comment:
The original sentence is fine to me and maybe most users, but it doesn't really explain what "contains" is. The alternative does that, but is ever so slightly more verbose. So, I'm not sure.

Reply:
I would say that 'contains' is clear as is. Just from the name of the function, users should pick up on the concept.

Reply:
No need to be verbose IMO

Reply:
For me there's a slight need to, but mostly because I want it to translate appropriately, too. On the other hand, the concept of "to have" or "contain" something is pretty dang global.